src/main/java/edu/ucla/library/prl/harvester/Config.java

```diff
@@ -71,6 +71,36 @@ public final class Config {
      */
     public static final String LDAP_URL = "LDAP_URL";
 
+    /**
+     * The ENV property for the LDAP authentication query.
+     */
+    public static final String LDAP_AUTH_QUERY = "LDAP_AUTH_QUERY";
+
+    /**
+     * The ENV property for the LDAP user query.
+     */
+    public static final String LDAP_USER_QUERY = "LDAP_USER_QUERY";
+
+    /**
+     * The ENV property for the attribute name used to authorize the user.
+     */
+    public static final String LDAP_ATTRIBUTE_KEY = "LDAP_ATTRIBUTE_KEY";
+
+    /**
+     * The ENV property for the attribute value used to authorize the user.
+     */
+    public static final String LDAP_ATTRIBUTE_VALUE = "LDAP_ATTRIBUTE_VALUE";
+
+    /**
+     * The ENV property for the LDAP user's username.
+     */
+    public static final String LDAP_USERNAME = "LDAP_USERNAME";
+
+    /**
+     * The ENV property for the LDAP user's password.
+     */
+    public static final String LDAP_PASSWORD = "LDAP_PASSWORD";
+
```

Comment:
I may be misunderstanding(?), but these are ENVs (not test specific) so will be set in the container to configure the application's use of the production LDAP server.

Reply:
Seems these are a result of some drift. Since these are only used in the tests I would prefer to move them to a class in the test package rather than in main.